src/main/java/Duke.java

```diff
@@ -14,36 +16,66 @@ public static void main(String[] args) {
         System.out.println(greeting);
 
         while(true){
-            String input = sc.nextLine();
+            String userInput = sc.nextLine();
             System.out.println("----------------------------------------------");
-            if(input.equals("bye")){
+            if(userInput.equals("bye")){
                 System.out.println(bye);
                 break;
             }
-            else if(input.equals("list")){
+            else if(userInput.equals("list")){//show list
                 System.out.println("Here are the tasks in your list:");
                 for(int i = 0; i<count ; i++)
-                    System.out.println((i + 1) + ".[" + tasks[i].getStatusIcon() + "] " + tasks[i].description);
+                    System.out.println(i+1 + "." + tasks[i]);
             }
-            else if(input.split(" ")[0].equals("mark")){
+            else if(userInput.split(" ")[0].equals("mark")){//mark task
```

Comment:
Perhaps you can consider using a boolean to reduce the complexity of this if condition? I think this can apply for the other if conditions within this while loop :)